seed/challenges/04-video-challenges/jslingo.json

```diff
@@ -63,7 +63,7 @@
       "description": [
         "We are going to cover what constitutes a variable, and the reasoning behind camelCase.",
         "A variable, also referred to as 'var', is the name or placeholder for a boolean, string, number, or other piece of static information.",
-        "You can use google dev tools to inspect the Free Code Camp home page and look for some variables.",
+        "You can use Google devtools to inspect the Free Code Camp home page and look for some variables.",
```

Comment:
Whoops! I think I was fixing things to 'devtools' but then found out it is written like 'Dev Tools' and missed this one as I was making things to 'Dev Tools'. I checked the rest of the files for any leftover 'dev tools' and there shouldn't be any. Thanks for the thorough double check!